pysquirrel/core.py

```diff
@@ -111,37 +114,34 @@ def _load(self) -> None:
         """
         Reads data from NUTS spreadsheet into Database and builds search index.
         """
+        urlretrieve(FILE_URL, FILENAME)
```

Comment:
For the future I would replace this with pooch (https://www.fatiando.org/pooch/latest/index.html). 
The advantage of this package is that it only downloads the file if necessary. The logic is that it checks if the file is present locally and if it is, it compares it to a known hash. 
The download is only activated if the file is not present or if the hashes don't match, i.e. the local file has been modified.
I'd save that change for a future PR though.